src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../App';
+
+type Props = {
+  todos: Todo[];
+  updateTodoList: React.Dispatch<React.SetStateAction<Todo[]>>;
+};
+
+export const TodoForm: React.FC<Props> = ({ todos, updateTodoList }) => {
+  const [currentUserId, setCurrentUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [isButtonClicked, setIsButtonClicked] = useState(false);
+
+  const addedTodoId = Math.max(...todos.map(todo => todo.id)) + 1;
```

Comment:
The calculation of `addedTodoId` assumes that `todos` is not empty. If `todos` is empty, `Math.max` will return `-Infinity`, leading to an incorrect `addedTodoId`. Consider handling the case where `todos` might be empty.